tests/testthat/test-simulate.R

```diff
@@ -36,7 +36,7 @@ test_that("simulate multi-state model", {
   y0 <- matrix(c(1000, 10, 0), 3, np)
   steps <- seq(0, 200, by = 20)
 
-  ans <- dust_simulate(res, steps, data, y0)
+  ans <- dust_simulate(res, steps, data, y0, seed = 1L)
```

Comment:
Do we check that two calls to dust_simulate give different answers with `seed = NULL` somewhere?

Reply:
There's a check here: https://github.com/mrc-ide/dust/blob/i87-rng-seed/tests/testthat/test-rng.R#L323-L335

Reply:
oh, not for simulate specifically. I'll add that though as we mention it in the docs

Reply:
added in 3d44d55